src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to={`/`}>Home</Link>
         </div>
         <div>
-          <a href="">About</a>
+          <Link to={'/about'}>About</Link>
         </div>
         <div>
-          <a href="">Contact</a>
+          <Link to={'/contact'}>Contact</Link>
         </div>
```

Comment:
while the { } don't break it here, unless you need to interpolate a value inside the template string (the backticks) you can safely omit them. the Links to parameters can look like this and still work also: to='/'     to='/about'    to='/contact'